addons/html_editor/static/src/utils/sanitize.js

```diff
@@ -58,7 +58,7 @@ export function initElementForEdition(element, options = {}) {
     }
 }
 
-export function fixInvalidHTML(content) {
+export function convertSelfClosingTagsToOpenClosed(content) {
```

Comment:
Why did you rename the function? Self-closing tags are not valid in html (they are in xml), the current name is correct.

Reply:
I see, but this function was specifically named that way so we could add other conversions to it in the future if needed without having to rename the function. In any case, even if I did agree with the renaming, it is an exported function in stable so we can't rename it anyway because other customers or partners could have imported it to use it in their own module. Honestly, I don't think they did, but they could, and we're not supposed to take the risk to break their code unless we have a very good reason to do so.

Reply:
makes sense.